internal/txlib/utils.go

```diff
@@ -210,3 +210,12 @@ func isValidResolutionPolicy(policy string) (IsValid bool) {
 	return false
 
 }
+
+func truncateMessage(message string) string {
+	const maxLength = 75
+
+	if len(message) > maxLength {
+		return message[:maxLength] + ".."
```

Comment:
Minor: You could do

```suggestion
		return message[:maxLength - 2] + ".."
```

here. The reasoning is this: Assume max length is 5. Then:

```
abc    -> abc
abcd   -> abcd
abcde  -> abcde
abcdef -> abc..
```